npm/repositories.bzl

```diff
@@ -1,11 +1,16 @@
 """Repository rules to fetch third-party npm packages"""
 
+load("@bazel_features//:deps.bzl", "bazel_features_deps")
 load("//npm/private:npm_import.bzl", _npm_import = "npm_import")
 load("//npm/private:npm_translate_lock.bzl", _list_patches = "list_patches", _npm_translate_lock = "npm_translate_lock")
 load("//npm/private:pnpm_repository.bzl", _LATEST_PNPM_VERSION = "LATEST_PNPM_VERSION", _pnpm_repository = "pnpm_repository")
 
 npm_import = _npm_import
-npm_translate_lock = _npm_translate_lock
+
+def npm_translate_lock(**kwargs):
+    bazel_features_deps()
```

Comment:
this is a bit awkward, but allows us a place to create the `bazel_features_versions` codegen repo without a breaking change for WORKSPACE users.
We assume everyone calls this at least once. If they don't, they'll have to call `bazel_features_deps()` themselves.
Sadly I don't know of a way to warn them about that when they get the error from the `load` statement.

Reply:
I added to the "workspace snippet" we publish on releases, so at least it's documented somewhere.